scripts/rtma3d/exrtma3d_gsianl.ksh

```diff
@@ -682,7 +682,7 @@ if [ "${envir}" == "lsf" ] || [ "${envir}" == "pbspro" ]; then #wcoss
     ./current_bad_aircraft ./gsd_sfcobs_uselist.txt ./gsd_sfcobs_provider.txt ./stdout*
   ${CP} -p  misc_info.tgz                      ${COMOUTgsi_rtma3d}
   gzip ${COMOUTgsi_rtma3d}/diag_*
-
+  ${CP} -p filelist.hrrrdas 		       ${COMOUTgsi_rtma3d}
```

Comment:
Hi @EdwardColon-NOAA,
Could you please also copy and archive the "filelist03" file? Actually "filelist03" is the file used by GSI. When using HRRRDAS ensemble, filelist.hrrrdas was generated first, then is copied to filelist03 for GSI to read; when using GDAS ensemble, only filelist03 is generated and used. 
So you may save both filelist03 and filelist.hrrrdas if convenient, at least the filelist03.

And please do the same thing in the analysis script for Alaska domain (akrtma3d/exakrtma3d_gsianl.ksh?) . actually for Alaska domain, no HRRRDAS ensemble, so no filelist.hrrrdas, only filelist03.

Thank you!

-Gang